src/main/java/com/sfuentes/CarLicensePlate.java

```diff
@@ -0,0 +1,48 @@
+package com.sfuentes;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CarLicensePlate {
+  String[] licensePlates;
+
+  public CarLicensePlate(String[] licensePlates) {
+    this.licensePlates = licensePlates;
+  }
+
+  public Map<String, Integer> getCountOfEachColorByLicensePlate(String[] licensePlates) {
```

Comment:
A suggestion to rename this method:
getLicenseCountByColor

Reply:
fixed